test/functional/epic.js

```diff
@@ -247,6 +248,37 @@ describe('quill', function () {
   });
 });
 
+describe('List copy/pasting into table', function () {
+  it('Should be no errors when list pasted into table cell(T1155500)', async function () {
+    // Test can be red on Mac because https://github.com/puppeteer/puppeteer/issues/1313
+    const tableCellSelector = '[data-table-cell="2"]';
+    const browser = await puppeteer.launch({
+      headless: false,
+    });
+    const page = await browser.newPage();
+    const pressKeyWithShortkey = async (keyName) => {
+      await page.keyboard.down(SHORTKEY);
```

Comment:
I don't see where this variable is initialized. Shouldn't we pass it to the function?